src/nixos/users/kreyren/home/machines/tupac/home-configuration.nix

```diff
@@ -0,0 +1,154 @@
+{ config, pkgs, lib, aagl, aagl-unstable, unstable, ... }:
+
+let
+	inherit (lib) mkIf;
+in {
+	home.stateVersion = "24.05";
+
+	gtk.enable = true;
+
+	home.impermanence.enable = true;
+
+	programs.alacritty.enable = true; # Rust-based Hardware-accelarated terminal
+	programs.bash.enable = true;
+	programs.starship.enable = true;
+	programs.direnv.enable = true; # To manage git repositories
+	programs.git.enable = true; # Generic use only
+	programs.gpg.enable = true;
+	programs.firefox.enable = true;
+	programs.vim.enable = true;
+	programs.vscode.enable = true; # Generic use only
+
+	services.gpg-agent.enable = (mkIf config.programs.gpg.enable true);
+
+	nixpkgs.config.allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [
+		"vscode"
+		"etlegacy"
+		"etlegacy-assets"
+	];
+
+	home.packages = [
```

Comment:
Why not use `home.packages = with pkgs; [` here to avoid reusing pkgs.x, pkgs.y, pkgs.z?

Reply:
> Why not use `home.packages = with pkgs; [` here to avoid reusing pkgs.x, pkgs.y, pkgs.z? - @TanvirOnGH (https://github.com/NiXium-org/NiXium/pull/74#discussion_r1660034125)

Because NIXium is designed to be release-independent and we don't follow the nixpkgs's ideology of removing flexibility and functionality bcs it's not needed atm

So when the user wants a package release that is not the configuration-set release it would be annoying to change it from your proposed to the desired.